Fuzzy/Sets/RightSlopeSet.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Fuzzy.Sets
+{
+    public class RightSlopeSet : FuzzySet
+    {
+        private readonly double peakPoint;
+        private readonly double rightOffset;
+
+        public RightSlopeSet(double peakPoint, double rightOffset) : base(peakPoint)
+        {
+            if (rightOffset < 0) throw new ArgumentOutOfRangeException(nameof(rightOffset));
+
+            this.peakPoint = peakPoint;
+            this.rightOffset = rightOffset;
+        }
+
+        public override double CalculateMembership(double value = 0)
+        {
+            if (value < this.peakPoint) { return 1.0; }
```

Comment:
Als de value kleiner is dan de peakpoint wordt 1.0 teruggeven. Dat maakt dit een LeftShoulder. Alleen noem je het een RightSlope, is dit omdat de slope van de set rechts zit?